src/core/Akka/Configuration/ConfigurationFactory.cs

```diff
@@ -62,10 +61,12 @@ public static Config ParseString(string hocon)
         /// <returns>The configuration defined in the configuration file.</returns>
         public static Config Load()
         {
-            var section = (AkkaConfigurationSection)ConfigurationManager.GetSection("akka") ?? new AkkaConfigurationSection();
-            var config = section.AkkaConfig;
-
-            return config;
+#if CONFIGURATION
+            var section = (AkkaConfigurationSection)System.Configuration.ConfigurationManager.GetSection("akka") ?? new AkkaConfigurationSection();
+            return section.AkkaConfig;
+#else
+            return ConfigurationFactory.Empty;
```

Comment:
For compatibility reasons ConfigurationFactory.Load just returns empty config on NetCor